system/system_monitor/src/cpu_monitor/cpu_monitor_node.cpp

```diff
@@ -33,20 +32,19 @@
 #else
 #include <system_monitor/cpu_monitor/unknown_cpu_monitor.h>
 #endif
+#include <system_monitor/utils.hpp>
 
 int main(int argc, char ** argv)
 {
-  ros::init(argc, argv, "cpu_monitor");
-  ros::NodeHandle nh;
-  ros::NodeHandle pnh("~");
+    rclcpp::init(argc, argv);
```

Comment:
[MINOR]: I think the indenting here and in other main files are inconsistent with the rest of package

Reply:
Restored back to the original indent.